roles/vald/defaults/main.yml

```diff
@@ -1,16 +1,16 @@
 ---
 # Maintains directories and paths
-vald_home_dir: /opt/{{ service_identifier }}
+vald_home_dir: "/opt/{{ service_identifier }}"
 
 vald_bin_dir: "{{ vald_home_dir }}/bin"
-vald_env_dir: "{{ vald_home_dir }}/env"
+vald_env_dir: "{{ vald_home_dir }}/environment"
+vald_vald_dir: "{{ vald_axelar_home }}/vald"
 vald_log_dir: "{{ vald_home_dir }}/log"
 
 vald_directories:
-  - "{{ vald_home_dir }}"
-  - "{{ vald_bin_dir }}"
-  - "{{ vald_env_dir }}"
-  - "{{ vald_log_dir }}"
+  - "{{ vald_vald_dir }}"
```

Comment:
might need to add the other directories in vald_directories too right ?

vald_bin_dir: "{{ vald_home_dir }}/bin"
vald_env_dir: "{{ vald_home_dir }}/environment"
vald_log_dir: "{{ vald_home_dir }}/log"